### PR TITLE
Adds StaffPhotographer Parser

### PR DESCRIPTION
**Pull request by kenoir**
Depends:
- https://github.com/guardian/grid/pull/914

### CODE REVIEW

**Comment by theefer** [comment]
Some tests please? :pray:

File: common-lib/src/test/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessorsTest.scala [hunk 1]
```diff
@@ -20,6 +20,14 @@ class SupplierProcessorsTest extends FunSpec with Matchers with MetadataHelper {
     processedImage.metadata.credit should be (Some("Unknown Party"))
   }
 
+  describe("StaffPhotographer") {
+    it("should match StaffPhotographer byline") {
+      val image = createImageFromMetadata("byline" -> "Christopher Thomond")
```
Comment (kenoir) [inline common-lib/src/test/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessorsTest.scala]: I'm a little bit uncomfortable about testing this hard-coded data, but as at the moment it _is_ in the code I feel this is acceptable, (and easier than mocking).

Comment (theefer) [inline common-lib/src/test/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessorsTest.scala]: :+1:

**Comment by theefer** [comment]
:+1: 

Let's add a data-debt entry for applying this to all the photographers on this list for existing images?

**Comment by kenoir** [comment]
Added data-debt entry: https://github.com/guardian/grid-infra/commit/3f84e4ad6b21a0d84e3cf8891689f45876cdd714

**Pull request closed without merging**